public/js/submit_delete_request_data.js

```diff
@@ -72,9 +72,14 @@ $(document).ready(function() {
         });
     });
 
-    // DELETE request when deleting observation
```

Comment:
why did this part get deleted? it's separate from the data submission and data request

Reply:
When I pulled a moment ago, I deleted all the changes I had made.... sooooo uhhhhhh, not sure.  You can delete this request

Reply:
fixed